src/components/Icons/Icons.constants.ts

```diff
@@ -3,3 +3,5 @@ import type { IconProps } from './Icons.types';
 export const DEFAULT_LABEL: IconProps['label'] = 'icon';
 export const DEFAULT_COLOR: IconProps['color'] = 'currentColor';
 export const DEFAULT_SIZE: IconProps['size'] = 24;
+export const DEFAULT_WIDTH: IconProps['width'] = 24;
+export const DEFAULT_HEIGHT: IconProps['height'] = 24;
```

Comment:
This is handled by `IconProps[`size`]` - we assume that all icons fit into a square container. Remove `width` and `height` and specify `size` instead.

Reply:
To match the design I have given these dimensions. I removed these as we are using IconCheck.